qemu/tests/cfg/live_snapshot.cfg

```diff
@@ -73,3 +73,11 @@
                 - stop_cont:
                     before_start = "stop"
                     after_finished = "resume reboot verify_alive"
+        - negative_test:
+            type = live_snapshot_negative_test
+            variants:
+            - @default:
```

Comment:
need indentation, otherwise occurs error: virttest.cartesian_config.ParserError: Expected ['variants', 'Identifier re([A-Za-z0-9][A-Za-z0-9_-]*)', 'only', 'no', 'include', 'del', '!', 'suffix', 'join'] got ['-']=[-]: '- existing_mode:' (/usr/share/avocado/data/avocado-vt/backends/qemu/cfg/subtests.cfg:1800)